src/components/Navbar.js

```diff
@@ -23,11 +24,12 @@ export default function Index() {
               <Nav.Link href="/">Home</Nav.Link>
               <Nav.Link href="/about">About</Nav.Link>
               <Nav.Link href="/services">Services</Nav.Link>
+              <Nav.Link href="/projects">Projects</Nav.Link>
               <Nav.Link href="/apply">Apply</Nav.Link>
               <Nav.Link
                 // Todo: Connect with Cheerio
-                // href="/contact"
-                href="https://forms.gle/uuA2s98v4oC4o8TU7"
+                href="/contact"
+                // href="https://forms.gle/uuA2s98v4oC4o8TU7"
```

Comment:
Yeah we may want to leave in this form in. We haven't added functionality from leadership to integrate Asana notifications when a client has reached us. This could be possibly another task we can work on.

Reply:
I will work on this while reorganizing the codes later. Thanks!